presto-main/src/main/java/com/facebook/presto/server/security/SpnegoFilter.java

```diff
@@ -223,6 +229,15 @@ private static void sendChallenge(HttpServletResponse response, boolean includeR
         response.setHeader(HttpHeaders.WWW_AUTHENTICATE, formatAuthenticationHeader(includeRealm, Optional.empty()));
     }
 
+    private static void skipRequestBody(HttpServletRequest request)
+            throws IOException
+    {
+        long contentLength = request.getContentLengthLong();
```

Comment:
Is this always trusted? Do we need to skip an exact number of bytes? What if we do this instead:
```java
ByteStreams.copy(request.getInputStream(), nullOutputStream());